eblocker-icapserver/src/main/java/org/eblocker/server/common/blacklist/CacheIndex.java

```diff
@@ -30,14 +31,21 @@
 
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 class CacheIndex {
+    @JsonIgnore
+    static final int CACHE_FORMAT = 5;
```

Comment:
If I have seen it correctly, you always create a CashIndex exactly from the current version using code. So it seems to me to be easier to put the constant into the class and always set the value. But not if it is read from a file.